test/unit/stream_alert_alert_processor/test_main.py

```diff
@@ -0,0 +1,100 @@
+'''
+Copyright 2017-present, Airbnb Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+'''
+import json
+
+from collections import OrderedDict
+from mock import patch, call
+
+from nose.tools import assert_equal
+
+from stream_alert.alert_processor.main import (
+    _load_output_config,
+    _sort_dict,
+    handler
+)
+
+from unit.stream_alert_alert_processor import (
+    REGION,
+    FUNCTION_NAME,
+    CONFIG
+)
+
+from unit.stream_alert_alert_processor.helpers import _get_mock_context
+
+
+class TestAlertProcessor(object):
+    """Test base class for Alert Processor"""
+    @classmethod
+    def setup_class(cls):
+        """Setup the class before any methods"""
+        pass
+
+    @classmethod
+    def teardown_class(cls):
+        """Teardown the class after all methods"""
+        pass
+
+@patch('logging.Logger.error')
```

Comment:
are you asking why the `@patch..` stuff is necessary? there are a bunch of instances of this in the tests, do you want them all commented?